app/models/table.rb

```diff
@@ -1239,6 +1254,10 @@ def qualified_table_name
     "\"#{owner.database_schema}\".\"#{@user_table.name}\""
   end
 
+  def qualified_remote_table_name
+    "\"commondata\".\"#{@user_table.name}\""
```

Comment:
Need a configurable way to get this. Does it make sense to just pull in `Cartodb::config[:fdw]['remote_schema']` here, or is there a better way to configure this?